frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx

```diff
@@ -125,6 +149,34 @@ const CampersTable = ({
     setCamperDetailsCount(tempDetailsCount);
   }, [campers]);
 
+  // Function to be called by the Move Campers Modal to update the campers
```

Comment:
Alert alignment should be more refined. It tends to hide when scrolling up and overlaps the delete button. I noticed that the error alert was not showing any message whenever the camper could not be moved. Another way to make the interaction better would be to reduce the time it's displayed on the page. Also, is there a specific reason for choosing the alert compared to using a toast (which also provides the user the functionality to close it whenever they like)?

<img width="770" alt="Screen Shot 2022-10-21 at 10 22 47 PM" src="https://user-images.githubusercontent.com/50252969/197344242-22f3dce8-5ae6-40c5-99af-bdc128afb44c.png">
<img width="770" alt="Screen Shot 2022-10-21 at 10 34 24 PM" src="https://user-images.githubusercontent.com/50252969/197344246-803af1f9-f147-46db-804c-9b54585931b1.png">